plugins/CoreHome/vue/src/AjaxHelper/AjaxHelper.ts

```diff
@@ -519,6 +522,12 @@ export default class AjaxHelper<T = any> { // eslint-disable-line
           resolve(data as (T | ErrorResponse)); // ignoring textStatus/jqXHR
         }
       }).fail((xhr: jqXHR) => {
+        if (xhr.status === 429) {
+          console.log(`Warning: the ${$.param(this.getParams)} rate limit!`);
```

Comment:
This shows something like:
`Warning: the module=Live&action=widget&disableLink=0&widget=1&idSite=1&period=range rate limit!`

It might be a bit clearer like this:
```suggestion
          console.log(`Warning: the '${$.param(this.getParams)}' request was rate limited!`);
```